en/micro-integrator/docs/administer-and-observe/logs/masking_sensitive_info_in_logs.md

```diff
@@ -1,52 +1,44 @@
 # Masking Sensitive Information in Logs
 
+!!! Info
+        The following feature is released as a product update on <b>11/11/2020</b>. If you don't already have this update, you can [get the latest updates](https://updates.docs.wso2.com/en/latest/updates/overview/) now.
+
 There can be business sensitive information that are added to logs. When these logs are analyzed, the information is exposed.
 
 To avoid this potential security pitfall, you can mask sensitive information (such as credit card numbers, access tokens, etc.) in the log file when the logs are created. You can also define patterns that need to be masked from the logs.
 
 ## Enabling log masking
 
-1.  Open the `MI_HOME/conf/log4j.properties` file in a text editor.
-2.  Uncomment or add the following property under `CarbonConsoleAppender` or `CarbonDailyRollingFileAppender`.  
+1.  Log masking in not enabled by default in Micro Integrator. Therefore, you need to manaully enable it and configure the required masking patterns.
+
+2.  To enable log masking, navigate to `<MI-HOME>/conf/log4j2.properties` and do the necessary changes. The masking feature can be enabled by adding an additional `m` after the `%m` in the `layout.pattern` parameter. Therefore you can add an additional `m` to the log file layout configuration in which you want the values to be masked as shown below.
 
-    ```xml
-    log4j.appender.CARBON_CONSOLE.maskingPatternFile=path-to-masking-patterns
     ```
+    appender.CARBON_CONSOLE.layout.pattern = [%d] %5p {%c{1}} - %mm%ex%n
+    ```
+3. The [masking patterns](#the-masking-pattern-file) are configured in `<MI-HOME>/conf/wso2-log-masking.properties` file. You can change its default configurations in `<MI-HOME>/conf/deployment.toml`.
```

Comment:
we don't need to mention wso2-log-masking.properties file since this is templated with deployment.toml file. I think this makes confuse to use both of the property files. WDYT?

Reply:
+1